Tests/NIOTests/AcceptBackoffHandlerTest.swift

```diff
@@ -96,7 +96,9 @@ public class AcceptBackoffHandlerTest: XCTestCase {
             return readCountHandler.readCount
         }.wait())
 
-        XCTAssertTrue(try serverChannel.pipeline.remove(name: acceptHandlerName).wait())
+        try serverChannel.pipeline.remove(name: acceptHandlerName).mapIfError { _ in
```

Comment:
Wasn't fully sure how to convert this, but figured this'd be the best way to go

Reply:
looks good